experiments/tootsie/exp600_tootsie.py

```diff
@@ -487,13 +503,17 @@
 
 llama_8b_train_config_phase4 = dataclasses.replace(
     llama_8b_train_config_phase3,
-    num_train_steps=PHASE_4_END,
+    num_train_steps=2_000_000,
```

Comment:
Why don't we use variables which are a bit more interpretable?

Reply:
sorry, this was a dumb thing i did to make pycharm refactor nicely